libsignal-service/Cargo.toml

```diff
@@ -33,13 +33,14 @@ rand = "0.7"
 
 uuid = { version = "0.8", features = [ "serde" ] }
 phonenumber = "0.3"
+hkdf = "0.11.0"
 
 [build-dependencies]
 prost-build = "0.8"
 
 [dev-dependencies]
 anyhow = "1.0"
-tokio = { version = "1.0", features = [ "macros" ] }
+tokio = { version = "1.0", features = [ "macros", "rt-multi-thread" ] }
```

Comment:
Why is this needed?

Reply:
will move the `actix` upgrade in another PR.

Reply:
This rt-multi-thread thing is still in. It doesn't affect the library itself, but it's in dev-deps and I don't see why? :'-)

Reply:
I don't mind this being here, but is it necessary somehow?